src/vm/exceptionhandling.cpp

```diff
@@ -17,6 +17,10 @@
 #include "eventtrace.h"
 #include "virtualcallstub.h"
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
```

Comment:
@parjong I would prefer to close this PR. I think it is easier to reason about ifdefs indicating this is architecture specific rather than about a new define.